### ISSUE TITLE
Overlays w/ transparency don't work in block environment  in pdf-/luaLaTeX

### ISSUE THREAD

**Issue by jakobherpich** [josephwright/beamer#387]
When I compile the following code with pdf- or luaLaTeX, the overlays do not work as expected:
```
\documentclass{beamer}

%\setbeamertemplate{blocks} % this works as expected
\setbeamertemplate{blocks}[rounded] % this only works properly with XeLaTeX

\setbeamercovered{transparent} % If uncommented everything works as expected but no transparent overlays
%\setbeamercovered{still covered={\opaqueness<1->{0}},again covered={\opaqueness<1->{20}}} % my original line, produces a similar problem

\begin{document}
\begin{frame}
    \begin{block}{Block 1}
        \begin{itemize}
            \item<+-1> item 1
        \end{itemize}
    \end{block}
    \begin{block}{Block 2}
        \begin{itemize}
            \item<+-> item 2
        \end{itemize}
    \end{block}
\end{frame}
\end{document}
```
This is the simplest example to reproduce the problem that I came up with. For more complex slides the behavior is very chaotic and overlays seem to behave very random.  The problem occurs only if I want round corners for the box environment  and transparent overlays.

If compiled with XeLaTeX, the problem goes away, everything works just fine.

Version information:

Beamer package (from `.log` file):
```
(/usr/share/texlive/texmf-dist/tex/latex/beamer/beamer.cls
Document Class: beamer 2017/01/22 v3.40 A class for typesetting presentations
```

```
$ pdflatex --version
pdfTeX 3.14159265-2.6-1.40.17 (TeX Live 2016/Debian)
kpathsea version 6.2.2
Copyright 2016 Han The Thanh (pdfTeX) et al.
There is NO warranty.  Redistribution of this software is
covered by the terms of both the pdfTeX copyright and
the Lesser GNU General Public License.
For more information about these matters, see the file
named COPYING and the pdfTeX source.
Primary author of pdfTeX: Han The Thanh (pdfTeX) et al.
Compiled with libpng 1.6.28; using libpng 1.6.28
Compiled with zlib 1.2.8; using zlib 1.2.8
Compiled with poppler version 0.48.0
```

```
$ lualatex --credits
This is LuaTeX, Version 0.95.0 (TeX Live 2016/Debian)

The LuaTeX team is Hans Hagen, Hartmut Henkel, Taco Hoekwater, Luigi Scarso.

LuaTeX merges and builds upon (parts of) the code from these projects:

tex       : Donald Knuth
etex      : Peter Breitenlohner, Phil Taylor and friends
omega     : John Plaice and Yannis Haralambous
aleph     : Giuseppe Bilotta
pdftex    : Han The Thanh and friends
kpathsea  : Karl Berry, Olaf Weber and others
lua       : Roberto Ierusalimschy, Waldemar Celes and Luiz Henrique de Figueiredo
metapost  : John Hobby, Taco Hoekwater and friends
poppler   : Derek Noonburg, Kristian H\ogsberg (partial)
fontforge : George Williams (partial)
luajit    : Mike Pall (used in LuajitTeX)

Compiled with libpng 1.6.28; using 1.6.28
Compiled with zlib 1.2.8; using 1.2.8
Compiled with poppler version 0.48.0
Compiled with mplib version 1.9991
```

```
$xelatex --version
XeTeX 3.14159265-2.6-0.99996 (TeX Live 2016/Debian)
kpathsea version 6.2.2
Copyright 2016 SIL International, Jonathan Kew and Khaled Hosny.
There is NO warranty.  Redistribution of this software is
covered by the terms of both the XeTeX copyright and
the Lesser GNU General Public License.
For more information about these matters, see the file
named COPYING and the XeTeX source.
Primary author of XeTeX: Jonathan Kew.
Compiled with ICU version 57.1; using 57.1
Compiled with zlib version 1.2.8; using 1.2.8
Compiled with FreeType2 version 2.6.3; using 2.6.3
Compiled with Graphite2 version 1.3.9; using 1.3.9
Compiled with HarfBuzz version 1.4.2; using 1.4.2
Compiled with libpng version 1.6.28; using 1.6.28
Compiled with poppler version 0.48.0
Compiled with fontconfig version 2.11.0; using 2.11.0
```

**Comment by josephwright** [comment]
Can you try with the latest release (v3.41)?

**Comment by jakobherpich** [comment]
I installed texlive through the Debian package manager so it is not straightforward to install the most recent version. Is there a standard way to temporarily install the most recent package version?

**Comment by louisstuart96** [comment]
<+-> specification works as an optional argument of the environment `itemize`.
This case should do correctly:

```
\documentclass{beamer}

\setbeamertemplate{blocks}[rounded]

\setbeamercovered{still covered={\opaqueness<1->{0}},again covered={\opaqueness<1->{20}}}

\begin{document}
\begin{frame}
    \begin{block}{Block 1}
        \begin{itemize}[<+-1>]
            \item item 1
        \end{itemize}
    \end{block}
    \begin{block}{Block 2}
        \begin{itemize}[<+->]
            \item item 2
        \end{itemize}
    \end{block}
\end{frame}
\end{document}
```

**Comment by jakobherpich** [comment]
This does not solve the problem. It also occurs when I use the `\uncover` command to display the individual items. I am quite convinced that this is a bug because it works as expected with XeLaTeX but not with pdf-/luaLaTeX.

**Comment by josephwright** [comment]
@jakobherpich  I'm not sure what you are referring to!

**Comment by jakobherpich** [comment]
I was referring to the previous post by @louisstuart96 .

**Comment by louisstuart96** [comment]
I tested it with LuaTeX/PDFTeX again, nothing is wrong.
Check versions of these packages/classes: `beamer.cls`, `xxcolor.sty`

**Comment by josephwright** [comment]
@louisstuart96 We already have that in the original post: v3.40 not the latest release v3.41.

**Comment by louisstuart96** [comment]
Is this the bug fixed in #355 ?

**Comment by jakobherpich** [comment]
@louisstuart96 This does indeed look very much like the same issue. I'm sorry for the duplicate. I assume the fix was introduced in v3.41. If that is the case, I will just use XeTeX until the Debian repositories upgrade to that version.
